ubuntu/standard/6.0/tools/runtime_configs/php/8.1.6

```diff
@@ -0,0 +1,20 @@
+configure_option "--with-curl"
+configure_option "--with-password-argon2"
+configure_option "--with-pdo-pgsql"
+configure_option "--with-libedit"
+
```

Comment:
Hi, please enable the sodium extension in 8.1 and 8.0 as well. refer to issue #538

Reply:
Hi, we won't be able to take this in for this release but will address this in a future release

Reply:
Still not updated